flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/data/DecimalDataTest.java

```diff
@@ -115,7 +115,7 @@ public void testNormal() {
         assertTrue(castToBoolean(castFrom(true, 5, 0)));
         assertEquals(5, castToIntegral(castFrom(5, 5, 0)));
         assertEquals(5, castToIntegral(castFrom("5", 5, 0)));
-        assertEquals(5000, castToTimestamp(castFrom("5", 5, 0)));
+        assertEquals(5000, castToTimestamp(castFrom("5", 5, 0)).getMillisecond());
```

Comment:
Could you add more unit tests for the new methods?

Reply:
ok